README.md

```diff
@@ -6,189 +6,25 @@
 
 ## Overview
 
-This project is a component of the [Operator Framework][of-home], an open source toolkit to manage Kubernetes native applications, called Operators, in an effective, automated, and scalable way. Read more in the [introduction blog post][of-blog].
-
-[Operators][operator_link] make it easy to manage complex stateful applications on top of Kubernetes. However writing an operator today can be difficult because of challenges such as using low level APIs, writing boilerplate, and a lack of modularity which leads to duplication.
-
-The Operator SDK is a framework that uses the [controller-runtime][controller_runtime] library to make writing operators easier by providing:
-- High level APIs and abstractions to write the operational logic more intuitively
+This project is a component of the [Operator Framework][of-home], an
+open source toolkit to manage Kubernetes native applications, called
+Operators, in an effective, automated, and scalable way. Read more in
+the [introduction blog post][of-blog].
+
+[Operators][operator_link] make it easy to manage complex stateful
+applications on top of Kubernetes. However writing an operator today can
+be difficult because of challenges such as using low level APIs, writing
+boilerplate, and a lack of modularity which leads to duplication.
+
+The Operator SDK is a framework that uses the
+[controller-runtime][controller_runtime] library to make writing
+operators easier by providing:
+- High level APIs and abstractions to write the operational logic more
+- intuitively
```

Comment:
I think that makes sense keep here just the overview info but not change it. 
Also, we should add the link for the web docs as well.  Which in POV means that it should be done just when the web docs be 100% ok and not in this PR.